src/Compatibility/Microsoft.DotNet.PackageValidation/Microsoft.DotNet.PackageValidation.csproj

```diff
@@ -6,7 +6,7 @@
     <IsShipping>true</IsShipping>
     <StrongNameKeyId>Open</StrongNameKeyId>
     <PackageType>MSBuildSdk</PackageType>
-    <VersionPrefix>0.1.0</VersionPrefix>
+    <VersionPrefix>1.0.0</VersionPrefix>
```

Comment:
why are we changing to 1.0.0?

Reply:
dont we want to ship the first package as 1.0.0-preview4 ?

Reply:
Hmm... yeah I don't have an strong opinion.